src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -267,20 +262,20 @@ def check_job_id__dne_test(self, request_type):
 
     def check_batch_id__no_job_test(self, request_type):
         for job_id in [None, ""]:
-            req_dict = make_comm_msg(request_type + "_batch", {JOB_ID: job_id}, False)
```

Comment:
all these requests are going from being `<request_type>_batch` with param `JOB_ID` to being `<request_type>` with param `BATCH_ID`